lib/Facebook.js

```diff
@@ -1182,16 +1182,25 @@ function Facebookbot(configuration) {
                 }
             });
         },
-        send: function(message_creative_id, custom_label_id, cb) {
+        send: function(message_creative, custom_label_id, cb) {
             var uri = 'https://' + api_host + '/' + api_version + '/me/broadcast_messages?access_token=' + configuration.access_token;
+            var body = {};
 
             if (facebook_botkit.config.require_appsecret_proof) {
                 uri += '&appsecret_proof=' + appsecret_proof;
             }
 
-            var body = {
-                'message_creative_id': message_creative_id
-            };
+            if (typeof message_creative === 'string') {
+                body = {
+                    'message_creative_id': message_creative
+                };
+            } else {
+                body = {
```

Comment:
It will be smart to clone all message_creative attributes into the body instead of copying one by one :

````
} else {
    body = message_creative
}
````